app/src/main/java/com/iitp/njack/iitp_connect/core/youtube/ChannelSelectActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.iitp.njack.iitp_connect.core.youtube;
+
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Toast;
+
+import com.iitp.njack.iitp_connect.R;
+import com.iitp.njack.iitp_connect.core.youtube.playlist.YoutubeActivity;
+import com.iitp.njack.iitp_connect.databinding.ActivityChannelSelectBinding;
+
+public class ChannelSelectActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        DataBindingUtil.setContentView(this, R.layout.activity_channel_select);
+
+    }
+
+    public void onChannelClick(View view) {
+        String id = getString(getResources().getIdentifier(getResources().getResourceEntryName(view.getId()), "string", getPackageName()));
+        Toast.makeText(this, id, Toast.LENGTH_SHORT).show();
```

Comment:
Define a showToast method in the `ViewUtils` file and use that.
File : https://github.com/Njack-IITP/IITP-Connect/blob/development/app/src/main/java/com/iitp/njack/iitp_connect/ui/ViewUtils.java